examples/order_posting_e2e.py

```diff
@@ -0,0 +1,120 @@
+# To run this test you will need to fill the .env file with the necessary variables (see .env.example).
+# You will also need to have enough funds in you wallet of the sell token to create the order.
+# The funds have to already be approved to the CoW Swap Vault Relayer
+
+import asyncio
+import json
+import os
+from dataclasses import asdict
+
+from web3 import Account
+
+from cow_py.common.chains import Chain
+from cow_py.common.config import SupportedChainId
+from cow_py.common.constants import CowContractAddress
+from cow_py.contracts.domain import domain
+from cow_py.contracts.order import Order
+from cow_py.contracts.sign import EcdsaSignature, SigningScheme
+from cow_py.contracts.sign import sign_order as _sign_order
+from cow_py.order_book.api import OrderBookApi
+from cow_py.order_book.config import OrderBookAPIConfigFactory
+from cow_py.order_book.generated.model import OrderQuoteSide1, TokenAmount
+from cow_py.order_book.generated.model import OrderQuoteSideKindSell
+from cow_py.order_book.generated.model import (
+    UID,
+    OrderCreation,
+    OrderQuoteRequest,
+    OrderQuoteResponse,
+)
+
+BUY_TOKEN = "0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14"  # WETH
+SELL_TOKEN = "0xbe72E441BF55620febc26715db68d3494213D8Cb"  # USDC
+SELL_AMOUNT_BEFORE_FEE = "10000000000000000000"  # 100 USDC with 18 decimals
+ORDER_KIND = "sell"
+CHAIN = Chain.SEPOLIA
+CHAIN_ID = SupportedChainId.SEPOLIA
+
+config = OrderBookAPIConfigFactory.get_config("prod", CHAIN_ID)
+ORDER_BOOK_API = OrderBookApi(config)
+
+ADDRESS = os.getenv("USER_ADDRESS")
+ACCOUNT = Account.from_key(os.getenv("PRIVATE_KEY"))
```

Comment:
Could we remove `ADDRESS` and simply use `ACCOUNT` for everything instead? 
I assume the sender/receiver can also act as signer.